charts/testrunnerdashboard/templates/service-account.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.name }}-service-ksa
```

Comment:
We typically standardize to `{{ .Values.name }}-sa` for k8s SA names

Reply:
Updated k8s service account name refs in all related manifests.